docs/content/en/schemes/local.md

```diff
@@ -139,6 +139,7 @@ auth: {
     local: {
       token: {
         required: false,
+        global: true,
```

Comment:
I believe this should be added in the example above, as this one is for cookie flow which doesn't use token.